test/suites/greenboot.robot

```diff
@@ -0,0 +1,109 @@
+*** Settings ***
+Documentation       Tests related to greenboot
+Resource            ../resources/common.resource
+Resource            ../resources/systemd.resource
+Resource            ../resources/ostree-health.resource
+Resource            ../resources/microshift-config.resource
+Library    SSHLibrary
+Suite Setup         Setup Suite 
+Suite Teardown      Teardown Suite
+
```

Comment:
Since this triggers MicroShift restarting, please apply the tags "restart" and "slow". See https://github.com/openshift/microshift/blob/main/test/suites/etcd.robot#L13 for an example.